src/libsync/theme.cpp

```diff
@@ -36,6 +36,22 @@
 #undef Mirall
 #endif
 
+namespace {
+
+QUrl imagePathToURL(const QString& imagePath)
```

Comment:
Nitpicks: URL -> Url, and the space should be before the & not after.

Qt coding style and all that.